sources/Outputs/include/ParGenerator.h

```diff
@@ -145,6 +134,25 @@ class ParGenerator {
   boost::shared_ptr<parameters::ParametersSet> writeGenerator(const algo::GeneratorDefinition& def, const std::string& basename,
                                                               const boost::filesystem::path& dirname);
 
+  /**
+   * @brief determine value of kGover based if a generator has a targetP equal to 0
```

Comment:
a boolean named fixedP is not clear enough name in my humble opinion. Why not directly taking as parameter the targetP as double and testing if its equal to 0 inside the method?

Reply:
I have changed it to take targetP as parameter, thx!